src/components/Registration/BusinessTypeConfirm.vue

```diff
@@ -1,32 +1,36 @@
 <template>
   <div id="business-type-confirm">
-    <v-row no-gutters>
-      <v-col cols="12" sm="3" class="pr-4">
-        <label class="d-block">Business Type</label>
-        <v-chip
-          v-if="hasBusinessTypeChecked"
-          id="checked-chip"
-          x-small label
-          color="primary"
-          text-color="white"
-        >
-          Checked
-        </v-chip>
-      </v-col>
-
-      <v-col cols="12" sm="9">
-        <p class="mb-0 checkbox-gp-text">{{ label }}</p>
-        <div id="business-check-div">
-          <v-checkbox
-            class="mt-0"
-            v-model="checked"
-            hide-details
-            :label="text"
+    <div class="section-container">
+      <v-row no-gutters>
+        <v-col cols="12" sm="3" class="pr-4">
+          <label class="d-block">Business Type</label>
+          <v-chip
+            v-if="hasBusinessTypeChecked"
+            id="checked-chip"
+            x-small label
+            color="primary"
+            text-color="white"
           >
-          </v-checkbox>
-        </div>
-      </v-col>
-    </v-row>
+            Checked
+          </v-chip>
+        </v-col>
+
+        <v-col cols="12" sm="9">
+          <p class="mb-0">{{ label }}</p>
+          <div id="business-checkbox-div">
+            <v-checkbox
+              hide-details
+              v-model="checked"
+              class="mt-0 pt-0"
+            >
+              <template slot="label">
+                <div class="certify-stmt" :class="{'error--text': showErrors}">{{ text }}</div>
```

Comment:
"certify-stmt" class is used to align with the "Certify" component. "error--text" class is added for unit test. div tag will have the class "error--text" if the checkbox is not checked upon reviewing.

Reply:
But did UI/UX ask for this? This component has a separate label above the checkbox so maybe the design for it is different.

Reply:
This component is the same as that in Certify component in bcrs. It handles the error color automatically. I also need this component for the unit test. The class "error--text" is added when the checkbox is not checked. If we use the implementation before with :label="text", then it is slightly more complicated. At the end of the day, UXA requires that this component uses the same  font size from "certify-stmt".

Reply:
My mistake. I was thinking about the label not the checkbox. This is correct.

Reply:
It's all good.